### PR TITLE
dep: updates wazero to v1.0.0-pre.1

### PR DESCRIPTION
**Pull request by mathetake**
Signed-off-by: Takeshi Yoneda <takeshi@tetrate.io>

### CODE REVIEW

File: pi-zero-ssr/handler/main.go
Comment (mathetake) [inline pi-zero-ssr/handler/main.go:4]: This file has been formatted with `gofmt` which is the official standard that everyone uses in Go projects.

**Review by ereslibre — APPROVED** [review]
Thanks @mathetake!

Approving, can be merged when the contributor license agreement is confirmed.

File: pi-zero-ssr/handler/go.mod
Comment (ereslibre) [inline pi-zero-ssr/handler/go.mod:5]: 🎉

**Review by Angelmmiguel — CHANGES_REQUESTED** [review]
Hello @mathetake !

Thank you very much for the pull request. Happy to see Wazero is close to v1.0.0! I already tested the new version on the Raspberry Pi Zero W and it works perfectly.

Before merging this PR, I found your editor modified all the spaces in the `main.go` file. To keep the MR only with required changes, please could you revert those space changes and keep only the one in the `import` block? (I think that's the only required)

We will lint these files and add a GitHub Action in a separate MR.

Thank you!

**Pull request merged**
